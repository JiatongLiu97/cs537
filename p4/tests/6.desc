[8] Test seg fault is still handled normally with CLOCKSIZE = 8
