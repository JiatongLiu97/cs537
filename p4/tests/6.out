XV6_TEST_OUTPUT TEST PASS
